### ISSUE TITLE
please remember to increase versionCode for each release

### ISSUE THREAD

**Issue by IzzySoft** [husudosu/mpv-remote-app#5]
First thanks for making your app available under a FOSS license and outside the usual "walled gardens"! May I kindly ask not to forget to increase `versionCode` before making a release? That's what Android uses to tell versions apart. Your latest release (1.0.5) still identifies with `versionCode: 1` – so it cannot be announced as "update" to users having a previous version already installed. Thanks in advance!

**Comment by husudosu** [comment]
So that's causes some of the updating problems. I thought increasing of versionName it's enough. I'm still learning about Android development and Ionic documentation, forums sometimes gives outdated, improper info. I've to dig in into Google's android documentation too. 
Thanks for the feedback!
Have a nice day!

**Comment by husudosu** [comment]
Thanks for the explanation and sharing my app via your repo! Of course I'm gonna place the badge on my repo.

The app id seems a bit off for me (I did not change that either), it shouldn't  be "io.ionic.starter" that's the default app id for Ionic. This probably gonna cause some problems, app id should be unique. For next release I gonna change something like this: 
"com.husudosu.mpvremote"

**Comment by IzzySoft** [comment]
Ah, I was already wondering about the applicationId :smile: My updater should then complain to me that the package name does not match, and I will rename it then (for which I have to remove the current release with the `io.ionic.starter` name then). Cannot hurt if you let me know here when the update is available, so I remember to "report success" to you :smiley:

**Comment by husudosu** [comment]
I've released [1.0.6](https://github.com/husudosu/mpv-remote-app/releases) version of the app. I've changed app id to ```com.husudosu.mpvremote``` and of course increased the versionCode variable too. Let me know if the app available on your repo with the new app id and gonna share it on my repo!
Thanks!

**Comment by IzzySoft** [comment]
Thanks for the note! I've just manually replaced the original package (which has been up for less than 2 weeks so I simply renamed everything, dropped the original APK, and manually put in the new one), so the "transition" should take place with the next sync around 19:00 UTC today. The link will then change according to the new package name: https://apt.izzysoft.de/fdroid/index/apk/com.husudosu.mpvremote – feel free to use a badge [available here](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) to link there e.g. from your Readme :smiley:
